packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -122,7 +122,7 @@ const AccountSwitcher = ({
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
-        RudderStack.setAccountType(loginid.substring(0, 2));
+        Analytics.setAttributes({ account_type: loginid.substring(0, 2) });
```

Comment:
this is while we are switching the account, so if we are switching the account, the account_type will change, so I think this is correct since this is updating the recent account_type

Reply:
I think this is fine and also in the other places as well, the values are getting changed so it should be updated for our rudderstack instance

Reply:
I believe this place can remove 2 another where we set them separately